### PR TITLE
First suggestion

### PR DESCRIPTION
**Pull request by eiriksm**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# References <!-- Link to the Jira task -->
```
Comment (esolitos) [inline pull_request_template.md]: I'd keep this a bit more low key than a `h1`. 😅 

```suggestion
**References** <!-- Link to the Jira task -->
```

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+[PROJ-XXX](https://frontkom.atlassian.net/browse/PROJ-XXX)
```
Comment (esolitos) [inline pull_request_template.md]: I think we should use the auto-links, so we can keep this simple:

```suggestion
PROJ-XXX
```

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+### Visuals <!-- Add here some screenshots -->
+
+### How can this be validated? <!-- Instructions for QA -->
+
+### Pull request checklist
+- [ ] Is it possible to deploy automatically? If it requires manual deployment, did you include the steps needed to deploy?
+- [ ] Did you make sure documentation have been updated, if needed?
+- [ ] Did you include all necessary translations, if any?
+- [ ] Did you make sure to update sanitization routines if fields for storing personal information are added?
```
Comment (esolitos) [inline pull_request_template.md]: I like shorter texts in there, faster to read and therefore more likely ppl will actually spend 2 sec reading them. 😄 

```suggestion
- [ ] Can be deploy automatically? _If manual actions are required:_ did you describe the deploy steps?
- [ ] Is the documentation updated, if it makes sense? <!-- Check this also if no updating is needed -->
- [ ] Are necessary translations added/updated? <!-- Check this even if no translations are being changed -->
- [ ] Did you update sanitization routines, where personal information is handled?
```

**Review by esolitos — COMMENTED** [review]
small changes suggested

File: pull_request_template.md (hunk 2 above)
Comment (eiriksm) [reply pull_request_template.md]: will it really auto link though?

File: pull_request_template.md (hunk 1 above)
Comment (xavierbarbosa) [reply pull_request_template.md]: Ha yes, I would like this too.

File: pull_request_template.md (hunk 2 above)
Comment (esolitos) [reply pull_request_template.md]: If configured yes and in on the commit text as well. :) I've set it up on several projects:
image

File: pull_request_template.md (hunk 1 above)
Comment (eiriksm) [reply pull_request_template.md]: lol no idea how it ended up like that

**Pull request merged**